### ISSUE TITLE
Grid-based multi-action scene control widget.

### ISSUE THREAD

**Issue by nullpanda** [home-assistant/android#1193]
**Is your feature request related to a problem? Please describe.**

Not a problem but a frustration, currently I have to create an individual action widget (which is somewhat time consuming) for each scene that I would like to be accessible from the homepage, and there is no way to save the configuration of these widgets as a group.
**Describe the solution you'd like**

Allow the user to create a grid with a predefined number of rows and columns, then assign each button to a scene that is already defined within Home Assistant.
**Describe alternatives you've considered, if any**

Hue Pro and Hue apps already have similar widgets, but this obviously doesn't allow you to control any of your lights outside of Hue. Currently I have individual action widgets from HA that are set to turn on a scene for each of the scenes I use. The limitation of this is that it isn't easily duplicable, meaning that I have to manually create/configure each widget from HA live, and adding to a second device would take the full amount of time again.
**Additional context**

https://imgur.com/a/INjQxiF | ignore the center widget with the up/down power control as that is very different to make.

**Comment by dshokouhi** [comment]
The Hue app itself is also a good example.  The widget contains an edit icon.  As the user makes the widget larger or smaller they get control over how many buttons they get.  This sorta like a dynamic service call widget where as the user expands the widget more buttons get added to be used.  The app is notified when the grid gets larger so the user can add more buttons.  Great idea!

**Comment by nullpanda** [comment]
I believe this request is still valid and has not been added to the app. Hoping it will still be possible in the future.

**Comment by kongo09** [comment]
I agree that widgets are limited right now. Lovelace is not. Taking this feature request to the next level, maybe there could be a way to tag lovelace cards to be able to become widgets.

Design would happen as is in HA. Once tagged, the card would show in the widget list ready to be added to the android screen. As lovelace is responsive, resizing of the widget would work, too.

This approach would allow for very complex widgets but also for the described grid based control widget.

**Comment by dshokouhi** [comment]
> I agree that widgets are limited right now. Lovelace is not. Taking this feature request to the next level, maybe there could be a way to tag lovelace cards to be able to become widgets.
> 
> Design would happen as is in HA. Once tagged, the card would show in the widget list ready to be added to the android screen. As lovelace is responsive, resizing of the widget would work, too.
> 
> This approach would allow for very complex widgets but also for the described grid based control widget.

widgets cannot use webview, they have to be built from the ground up....we cant jsut display lovelace cards like that as much as we would like.

**Comment by kongo09** [comment]
Maybe using an image instead of a webview could solve this?

https://stackoverflow.com/questions/33927617/android-loading-webview-output-into-app-widget

**Comment by nullpanda** [comment]
Hello, this is still an open request, the feature has not yet been added or given a close alternative. 

Thanks

**Comment by dnestico** [comment]
Would love a favorites widget just like the Google home app, even the IOS home assistant app just got newly updated widgets which look really good, hoping to see those on the Android app too some day...

**Comment by dshokouhi** [comment]
> even the IOS home assistant app just got newly updated widgets which look really good, hoping to see those on the Android app too some day...

please feel free to create a separate request for this and attach screenshots of how the widget looks in iOS for the team to consider

**Comment by dnestico** [comment]
Done. https://github.com/home-assistant/android/issues/4549
